cli/src/dolos.ts

```diff
@@ -10,11 +10,17 @@ import * as path from "path";
 import { Tokenizer } from "./lib/tokenizer/tokenizer";
 import { CharTokenizer } from "./lib/tokenizer/charTokenizer";
 import { default as fsWithCallbacks } from "fs";
+import { IPyNbTokenizer } from "./lib/tokenizer/ipynbTokenizer";
+import { MarkdownTokenizer } from "./lib/tokenizer/markdownTokenizer";
 const fs = fsWithCallbacks.promises;
 
 function newTokenizer(language: string): Tokenizer {
   if (language == "chars") {
     return new CharTokenizer();
+  } else if (language == "ipynb") {
+    return new IPyNbTokenizer();
+  } else if (language == "md") {
```

Comment:
The alternative here would be to make something like
```typescript
const tokenizers = new Map();
tokenizers.add("chars", () => new CharTokenizer());
tokenizers.add("ipynb", () => new IPyNbTokenizer());
...

const tokenizer = tokenizers.get(language);
if (tokenizer) {
  return tokenizer.call();
} else {
  return new CodeTokenizer(language);
}
```
But this does not really remove complexity. A switch would also make sense here because of the `default`.